addons/options/README.md

```diff
@@ -1,136 +1,28 @@
-See the [migration docs](../../MIGRATION.md#options-addon-deprecated) for what's changed.
-
-# Storybook Options Addon
-
-The Options addon can be used to (re-)configure the [Storybook](https://storybook.js.org) UI at runtime.
-
-[Framework Support](https://github.com/storybookjs/storybook/blob/master/ADDONS_SUPPORT.md)
-
-![Screenshot](https://raw.githubusercontent.com/storybookjs/storybook/HEAD/addons/options/docs/screenshot.png)
-
-## Getting Started
-
-First, install the addon
-
-```sh
-yarn add @storybook/addon-options --dev
-```
-
-Add this line to your `addons.js` file (create this file inside your storybook config directory if needed).
-
-```js
-import '@storybook/addon-options/register';
-```
-
-### Set options globally
-
-Import and use the `addParameters` + `options`-key in your `config.js` file.
+Global options can be specified with:
 
 ```js
-import { addParameters, configure } from '@storybook/react';
-
-// Option defaults:
-addParameters({
-  options: {
-    /**
-     * name to display in the top left corner
-     * @type {String}
-     */
-    name: 'Storybook',
-    /**
-     * URL for name in top left corner to link to
-     * @type {String}
-     */
-    url: '#',
-    /**
-     * show story component as full screen
-     * @type {Boolean}
-     */
-    goFullScreen: false,
-    /**
-     * display panel that shows a list of stories
-     * @type {Boolean}
-     */
-    showStoriesPanel: true,
-    /**
-     * display panel that shows addon configurations
-     * @type {Boolean}
-     */
-    showAddonPanel: true,
-    /**
-     * display floating search box to search through stories
-     * @type {Boolean}
-     */
-    showSearchBox: false,
-    /**
-     * show addon panel as a vertical panel on the right
-     * @type {Boolean}
-     */
-    addonPanelInRight: false,
-    /**
-     * display the top-level grouping as a "root" in the sidebar
-     * @type {Boolean}
-     */
-    showRoots: null,
-    /**
-     * sidebar tree animations
-     * @type {Boolean}
-     */
-    sidebarAnimations: true,
-    /**
-     * id to select an addon panel
-     * @type {String}
-     */
-    selectedPanel: undefined, // The order of addons in the "Addon panel" is the same as you import them in 'addons.js'. The first panel will be opened by default as you run Storybook
-    /**
-     * enable/disable shortcuts
-     * @type {Boolean}
-     */
-    enableShortcuts: false, // true by default
-    /**
-     * show/hide tool bar
-     * @type {Boolean}
-     */
-    isToolshown: false, // true by default
-  },
+import '@storybook/addon-roundtrip/register';
+import '@storybook/addon-parameter/register';
+import '@storybook/addon-preview-wrapper/register';
+
+import { addons } from '@storybook/addons';
+import { themes } from '@storybook/theming';
+
+addons.setConfig({
+  theme: themes.dark,
+  showNav: true,
+  showPanel: true,
+  panelPosition: 'bottom',
+  enableShortcuts: true,
+  isToolshown: true,
+  selectedPanel: 'storybook/roundtrip',
```

Comment:
What happened to the other options (e.g. `url`, `showAddonPanel` etc)?

Reply:
They are to be documented on the docs site instead.

I want to completely remove this addon.

Reply:
Ok, but are these options documented somewhere else?

Reply:
https://github.com/storybookjs/storybook/blob/651e058cee6cecbb8280da4025c8e56345c883a3/docs/src/pages/configurations/options-parameter/index.md

Reply:
I updated to complete remove this content and just refer to the docs.

This prevents these docs from going out of sync.